src/aind_analysis_arch_result_access/han_pipeline.py

```diff
@@ -276,6 +271,59 @@ def get_autotrain_table():
     return df_autotrain
 
 
+def get_docDB_table() -> pd.DataFrame:
+    """
+       Load the curriculum data from the behavior json in docDB
+    """
+
+    docdb_client = aind_data_access_api.document_db.MetadataDbClient(
+        host="api.allenneuraldynamics.org", database="metadata_index", collection="data_assets"
+    )
+    sessions = docdb_client.retrieve_docdb_records(
+        filter_query={"name": {"$regex": "^behavior_(?!.*processed).*"}}
+    )
+
+    df_dict = {
+        "subject_id": [],
+        "session_date": [],
+        "curriculum_name": [],
+        "curriculum_version": [],
+        "current_stage_actual": [],
+        "current_stage_suggested": [],
+        "if_overriden_by_trainer": [],
+        "next_stage_suggested": [],
+        "if_closed_loop": [],  # IMPORTANT: automatically set to True to merge with autotrain
+    }
+
+    for session in sessions:
+        try:
+            if curriculum_params := session["session"]["stimulus_epochs"][0][
+                "output_parameters"
+            ].get("streamlit"):
```

Comment:
The name "streamlit" sounds misleading. Should we change back to "curriculum" in all the places?